.github/workflows/kernel.yml

```
@@ -8,6 +8,13 @@ on:
       - 'package/kernel/**'
       - 'target/linux/generic/**'
 
+  push:
+    paths:
+      - '.github/workflows/kernel.yml'
+      - 'include/kernel-*'
+      - 'package/kernel/**'
+      - 'target/linux/generic/**'
+
```

```suggestion
  push:
    paths:
      - '.github/workflows/kernel.yml'
      - 'include/kernel-*'
      - 'package/kernel/**'
      - 'target/linux/generic/**'

```

leftover from testing I assume

---

If they are not welcome, I would remove them after my trick proves working.